simapp/app.go

```diff
@@ -20,6 +20,8 @@ import (
 	"github.com/cosmos/cosmos-sdk/x/auth"
 	"github.com/cosmos/cosmos-sdk/x/auth/ante"
 	"github.com/cosmos/cosmos-sdk/x/bank"
```

Comment:
Why is this import here still?

Reply:
The `NewAppModule` function is in the `bank` module.go. It is neither in `bank/types` nor in `bank/keeper`. Therefore we still need the import above (`"github.com/cosmos/cosmos-sdk/x/bank"`)
![Selection_065](https://user-images.githubusercontent.com/30392990/84595721-ce9dec80-ae59-11ea-9758-79ebc6096241.png)